src/BaseTemplates/EmptyWeb/project.json

```diff
@@ -30,7 +30,10 @@ $if$ ($context$ == WebCore) $else$
   },
 $endif$
   "tools": {
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
+      "version": "__AspNetCorePlatformVersion__",
+      "imports": "portable-net45+wp80+win8+wpa81+dnxcore50"
+    }
   },
 
```

Comment:
I'm assuming yes, due to other content files in wwwroot.   I imagine we still want to specify even for empty template for consistent behavior and customer convenience.



On Apr 18, 2016, at 4:33 PM, Pawel Kadluczka <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/BaseTemplates/EmptyWeb/project.json<https://github.com/aspnet/Templates/pull/512#discussion_r60152584>:

> @@ -30,7 +30,10 @@ $if$ ($context$ == WebCore) $else$
>    },
>  $endif$
>    "tools": {
> -    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
> +    "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
> +      "version": "__AspNetCorePlatformVersion__",
> +      "imports": "portable-net45+wp80+win8+wpa81+dnxcore50"
> +    }
>    },
>


do you still need wwwroot in the content after web.config was moved to the app root?

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/aspnet/Templates/pull/512/files/46c506bff95b7d383c07df3155e22abbe927eaa0#r60152584>